lib/imagemagick.js

```diff
@@ -15,7 +15,8 @@
 var sprintf = require('util').format,
   log = require('sslog'),
   lru = require("lru-cache"),
-  child = require('child_process');
+  child = require('child_process')
+  playButtonPath = __dirname + '/../public/play-button.svg';
```

Comment:
I was more talking about a local "config" file that maps assets to their paths than the one in the builds repo, but those are good points. I'm fine with leaving it like this 👍